tests/config/plugin/configuration.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Configuration file for use in testing

Reply:
We can maybe move the graphql config out into a directory:

```
config/
├── configuration.json
├── get_users/
│   ├── configuration.json
│   └── query.gql
└── users_by_key/
    ├── configuration.json
    └── query.gql
```

Reply:
We can do this later.